variables.tf

```
@@ -1,115 +1,115 @@
-locals {
-  base_tags = {
-    tf_module = "ecs-cluster"
-  }
-}
-
-variable "name" {
-  description = "common name for resources in this module"
-  type        = string
-  default     = "ecs-cluster"
-}
 
-variable "region" {
-  type = string
+variable "alb_subnet_ids" {
+  description = "Subnets ALB will listen on"
+  type        = list(string)
 }
 
-variable "tags" {
-  type        = map(string)
-  description = "common tags for all resources"
-  default     = {}
+variable "asg_health_check_type" {
+  default     = "EC2"
+  description = "Check instance health with EC2 or ELB checks"
+  type        = string
 }
 
-variable "instance_policy_document" {
-  description = "Policy document for instance IAM role"
+variable "asg_max_size" {
+  default     = 1
+  description = "Maximum batch size for ASG rolling updates"
   type        = string
-  default     = null
 }
 
 variable "custom_iam_policy" {
+  default     = false
   description = "Whether you're passing a custom policy document"
   type        = bool
-  default     = false
 }
 
+variable "desired_instances" {
+  default     = 2
+  description = "Desired instances in ASG"
+  type        = string
+}
 
-variable "alb_subnet_ids" {
-  description = "Subnets ALB will listen on"
-  type = list(string)
+variable "ec2_public_ip" {
+  default     = true
+  description = "Whether to assign a public IP to autoscaled instances"
+  type        = string
```

```suggestion
  type        = bool
```
bools should be bools